File: src/homework/Nov26/miniProject/Product.java
package homework.Nov26.miniProject;

public class Product{
	private int id;
	private String title;
	private long price;
//	private int amount;
	
	public Product(Integer id, String title, long price) {
		this.id = id;
		this.title = title;
		this.price = price;
//		this.amount = amount;
	}
	
	public int getId() {
		return id;
	}
	
	public String getTitle() {
		return title;
	}

	public long getPrice() {
		return price;
	}
	
//	public int getAmount() {
//		return amount;
//	}

	@Override
	public String toString() {
		return id + " " + title + " " + price + " Baht.";
	}
	
	
}

File: src/lab/miniProject_TS/src/product.ts
class Product {
    private id: number
    private title: string
    private price: number
    
    constructor(id: number, title: string, price: number) {
        this.id = id
        this.title = title
        this.price = price
    }

    public getId(): number {
        return this.id
    }

    public getTitle(): string {
        return this.title
    }

    public getPrice(): number {
        return this.price
    }

    public toString(): string {
        return `${this.id} ${this.title} ${this.price}`
    }

}

export default Product
